src/operator/config/webhook/manifests.yaml

```diff
@@ -32,6 +33,7 @@ webhooks:
       namespace: system
       path: /validate-k8s-otterize-com-v1alpha3-clientintents
   failurePolicy: Fail
+  matchPolicy: Exact
```

Comment:
@orishoshan - Without `match:Policy: Exact` the client ignores the `apiVersion` rule - so in our case, each ClientIntent is sent to every validation webhook